userspace/falco/falco_metrics.cpp

```diff
@@ -86,306 +99,346 @@ std::string falco_metrics::to_text(const falco::app::state& state) {
 			}
 		}
 
-		const scap_agent_info* agent_info = inspector->get_agent_info();
-		const scap_machine_info* machine_info = inspector->get_machine_info();
-		libs::metrics::libs_metrics_collector libs_metrics_collector(inspector.get(), 0);
-		prometheus_text += prometheus_metrics_converter.convert_metric_to_text_prometheus(
-		        "version",
-		        "falcosecurity",
-		        "falco",
-		        {{"version", FALCO_VERSION}});
-
-		// Not all scap engines report agent and machine infos.
-		if(agent_info) {
-			prometheus_text += prometheus_metrics_converter.convert_metric_to_text_prometheus(
-			        "kernel_release",
-			        "falcosecurity",
-			        "falco",
-			        {{"kernel_release", agent_info->uname_r}});
-		}
-		if(machine_info) {
-			prometheus_text += prometheus_metrics_converter.convert_metric_to_text_prometheus(
-			        "hostname",
-			        "falcosecurity",
-			        "evt",
-			        {{"hostname", machine_info->hostname}});
-		}
+		if(i == 0) {
+			// Falco wrapper metrics; only once
+			//
+
+			// Each inspector includes all event sources
+			/* Examples ...
+			    # HELP falcosecurity_falco_evt_source_info https://falco.org/docs/metrics/
```

Comment:
@sgaist and @FedeDP: We seem to have them in each inspector, hence we may only need to query the first inspector. While testing with the dummy_c evert sourc plugin this assumptions appears to hold true.